README.md

```diff
@@ -1,45 +1,103 @@
 # QUsb2snes
 
+[![License: GPL v3](https://img.shields.io/badge/License-GPLv3-blue.svg)](https://www.gnu.org/licenses/gpl-3.0)
+[![C/C++ CI](https://github.com/Skarsnik/QUsb2snes/actions/workflows/buildlinux.yml/badge.svg)](https://github.com/Skarsnik/QUsb2snes/actions/workflows/buildlinux.yml)
+[![Build status](https://ci.appveyor.com/api/projects/status/r8t2hpt21ux5r7mi/branch/master?svg=true)](https://ci.appveyor.com/project/Skarsnik/qusb2snes/branch/master)
+
 QUsb2Snes is a websocket server that provide an unified protocol for accessing hardware/software that act like a SNES (or are a SNES).
 A classic usage is to use the FileViewer client to upload roms to your SD2SNES.
 But it allows for more advanced usage like reading/writing the memory of the SNES.
 
-[![Build Status](https://travis-ci.com/Skarsnik/QUsb2snes.svg?branch=master)](https://travis-ci.com/Skarsnik/QUsb2snes)
-[![Build status](https://ci.appveyor.com/api/projects/status/r8t2hpt21ux5r7mi/branch/master?svg=true)](https://ci.appveyor.com/project/Skarsnik/qusb2snes/branch/master)
+## Installation
 
-## Developers
+If you want the full experience go [Usb2Snes project](https://github.com/usb2snes/usb2snes/releases).
+Otherwise, download the latest release of [QUsb2Snes](https://github.com/Skarsnik/QUsb2snes/releases).
 
-Look at the `docs/Protocol.md` file to have a detailed view of the command of websocket protocol.
+This repository only have QUsb2Snes and a file viewer application.
+You can also use the provided binaries from GitHub releases or [compile QUsb2Snes yourself from source](COMPILING.adoc).
+On Linux, some distributions already ship qusb2snes (like arch AUR).
+
+Installation via other means (homebrew, chocolatey, etc.) is not planned at the moment but may be provided in the future, if there is demand.
+
+---
 
 ## Usage
 
-This repository only have QUsb2Snes and a file viewer application. 
-If you want the full experience go [Usb2Snes project](https://github.com/usb2snes/usb2snes/releases).
-Otherwise, download the latest release of [QUsb2Snes](https://github.com/Skarsnik/QUsb2snes/releases).
+### Command line options
```

Comment:
The no gui should probably be at the bottom, it's like for 1-2% of the users x) Also I removed the -nogui option since it's irrelevant when compiling without the gui